src/components/Homepage/markdown-links-to-react-components.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const SURROUNDING_TEXT_BEFORE = 0;
+const EXTRACTED_LINK_TEXT = 1;
+const EXTRACTED_LINK_HREF = 2;
+const SURROUNDING_TEXT_AFTER = 3;
+
+export const markdownLinksToReactComponents = (textWithMarkdownLinks: string) => {
```

Comment:
What would be nice is if you could get rid of this whole class and replace it with:
https://www.npmjs.com/package/markdown-to-jsx
which _should_ just do it all.